docs/getting-started/governance.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Governance

```

Comment:
It's hard for me to justify this section when most (if not all) of the content is here:
 https://docs.umaproject.org/uma-tokenholders/uma-holders
 
UMA tokenholders voting on price requests is a part of the DVM (so we should talk about it there) versus having it's own section. @smb2796 what do you think on this section? I am open to being convinced otherwise.